### PR TITLE
Bump JBang to 0.110.1

### PR DESCRIPTION
**Pull request by jsmrcka**
Requested by Quarkus QE (`quarkus-qe-ts`), based on test failures, see e.g.
https://github.com/quarkus-qe/quarkus-test-suite/actions/runs/5947272576/job/16129106765.

### CODE REVIEW

**Review by gsmet — COMMENTED** [review]
Is it something we really want to install for all the ecosystem?

I'm not sure it is. And if so, it's probably better to have a custom `.github/ci-prerequisites.sh`.

**Comment by gastaldi** [comment]
@gsmet that makes sense. @jsmrcka would a custom `.github/ci-prerequisites.sh` in your repo performing the CLI installation work for you?

**Comment by jsmrcka** [comment]
@gsmet @gastaldi Yes, that makes more sense. I'll prepare an update.

**Comment by gsmet** [comment]
That being said I think we should have a way to add an initialization script and not replace the existing one, given the existing one is probably always useful.

**Comment by gsmet** [comment]
I'm closing this, will prepare a PR to be able to add a custom script and will ping you there.

**Comment by gsmet** [comment]
@jsmrcka so just add an executable `.github/ci-init.sh` to your repository and install the CLI there.

**Comment by jsmrcka** [comment]
> @jsmrcka so just add an executable `.github/ci-init.sh` to your repository and install the CLI there.

What about the `jbang` upgrade, is it worth changing in the main `setup-and-test` script? I can do without it, it's just nice to have.
@gsmet

**Comment by gsmet** [comment]
Yes, I think it's worth updating it in the main script. Please create a PR for that. Thanks!

**Comment by gastaldi** [comment]
Or we could reopen this PR and you can force push the commit containing the update :)

File: setup-and-test [hunk 1]
```diff
@@ -80,6 +80,16 @@ else
...
+echo "Installing Quarkus CLI"
+jbang app install --force --name quarkus "${HOME}/.m2/repository/io/quarkus/quarkus-cli/${QUARKUS_VERSION}/quarkus-cli-${QUARKUS_VERSION}-runner.jar"
+export PATH="${HOME}/.jbang/bin:${PATH}"
+if quarkus --version; then
+  echo "Quarkus CLI installed successfully"
+else
+  echo "Failed to install Quarkus CLI"
+  exit 1
+fi
+
```
Comment (gastaldi) [inline setup-and-test:92]:
```suggestion
```

**Pull request merged**